### DISCUSSION TITLE
Playing a chime without a message

### DISCUSSION THREAD

**Discussion by dimatx** [nimroddolev/chime_tts#112] [General]
I have some use cases where I want to play a chime without any message. I realize this is not the purpose of Chime TTS, but given it's features to join and unjoin speakers and others, I was hoping to rely on Chime TTS for this as well. Is it possible to have the 'message' be optional as long as a chime is provided?

**Comment by dbullendsl** [comment]
just use a single space in leu of any text

**Reply by dimatx** [reply]
I have tried that but it had caused issues with the TTS hanging, and preventing future announcements for some time.

**Comment by nimroddolev** [comment]
Hi @dimatx. No problem - I will make the `message` parameter optional in the next release 👍

**Reply by dimatx** [reply]
Awesome! Thanks for your contributions to HA with this great integration.

**Comment by nimroddolev** [comment]
Hi @dimatx. I just released a new beta, v1.1.0-beta3, which includes support for using Chime TTS without the `message` parameter, allowing you to use it with chimes only.

Please download the new version and let me know if it's working for you.

To install the new beta version:
1. Open the <a href="https://my.home-assistant.io/redirect/hacs_repository/?owner=nimroddolev&repository=chime_tts&category=integration" title="Chime TTS HACS repository" target="_blank">Chime TTS repository in HACS</a>
1. Click on the `⋮` button in the top right
1. Select the `↻ Redownload` option
1. Enable the `Show beta versions` option (it may take a moment to update the list)
1. Select version `v1.1.0-beta3` from the updated version drop-down list
1. Restart Home Assistant

**Reply by dimatx** [reply]
Thanks @nimroddolev! Will try it out today 😊

**Reply by dimatx** [reply]
Works as expected 😊

**Comment by nimroddolev** [comment]
👍 happy to hear it!
